taichi/ir/snode.h

```diff
@@ -154,61 +154,60 @@ class SNode {
 
   SNode &insert_children(SNodeType t);
 
-  SNode &create_node(std::vector<Index> indices,
+  SNode &create_node(std::vector<Axis> indices,
```

Comment:
1. Could you also help change the param name from `indices` to `axes`? (Thanks @strongoier for pointing this out)
2. https://github.com/taichi-dev/taichi/blob/f276de47518d900282ccaede5c1fe0a6f6170087/taichi/ir/snode.h#L32-L35 IndexExtractor also needs to be renamed to AxisExtractor